Cleans up details a bit. More compact, order more relevant. Fixes #35

plocket opened the pull request:
<!-- Thank you for submitting a pull request to the CFA response theme. Please provide information about the changes: What issue they fix, how they solve the issue, and why the solution works. -->

## Description
<!-- Add the issue number this pull request addresses: -->
Fixes #35
<!-- Please describe your pull request. -->
Shelter details should now look like this:
- address
- directions
- phone icon, phone number
- 'Pets' label, pets details
- Notes label
- Notes details

## List of changes
<!-- Please describe what was changed/added. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [x] I want my code added to the response theme.

plocket commented:
@miklb : What more does this need before it can be merged?

plocket commented:
@omnilord : I can't add reviewers, so I'll just ping you here for a review.

plocket commented:
@omnilord : done as far as I can tell

omnilord commented:
Looks decent.  Removing the directions arrow may be a bit confusing since it is a ubiquitous visual cue for "get me directions to this location." Thoughts on making it more stylistically a better fit kind of like with the phone icon?

plocket commented:
The arrow doesn't help during the day because it's white. I can change the color. Or show me the phone icon and I can try. Does this theme have fontawesome?

plocket commented:
@omnilord

miklb commented:
No, it uses SVGs. I have an undocumented gulp task to inline svgs Previously they were just hard coded.

miklb commented:
I might be thinking of another project though, I’ve done similar.

plocket commented:
Oh, shoot, I was thinking of a different PR from earlier. I can restore this one.

plocket commented:
Added back in. I also noticed it was a little hard to tell that the links were links. It's hard to see the blue color. At least, it is for me. I added underlines. Let me know if that's a problem.

miklb commented:
looks good to me. Probably want a new issue for the `null` on notes unless someone has a quick solution to add to this PR.

omnilord commented on assets/css/_inc/map.css:
```diff
@@ -177,6 +182,10 @@
     line-height: .7em;
 }
 
+#icon-directions {
```
Did you mean to leave this style empty?

plocket replied:
I did not! Thanks for catching that!

plocket commented:
@miklb : If you think it's related enough to this PR, I can take a stab. Let me know if it'd just be better to merge what we have so other merges can move forward.

miklb commented:
Not a blocker on my end. If you wouldn't mind opening the issue that would be helpful to remember. I'm working through some organizing now.

plocket commented:
@miklb : It looks like it was maybe taken care of? I'm getting 'Unknown' for a lot of these and not seeing any `null`s.

@miklb , @omnilord : Is this ready to merge?

plocket commented:
@miklb : Nvm, just found one I can test with!

plocket commented:
@miklb : I've figured out a way. I'll make an issue and then a PR once this other stuff has been merged.

omnilord commented on assets/css/main.css:
```diff
@@ -436,14 +436,21 @@ a:active {
     font-size: 2.5em;
     line-height: .7em;
+}
+#icon-directions {
```
Can I get you to do a `gulp build` then push the updated `main.css` and `main.css.map`?

plocket commented:
@omnilord : Done. Didn't know it didn't happen before and not sure why... a puzzle for another time.
